kemeny_sourcemash.tex

```diff
@@ -43,15 +43,24 @@ \section{Project Overview}
 
 \section{User Instructions}
 
-While
+Sourcemash is a web application, located at \url{www.sourcemash.com}. The source code for Sourcemash is available at \url{https://github.com/sourcemash/Sourcemash}.
 
-\subsection{Production: www.sourcemash.com}
+\subsection{Production}
 
-When
+Users are encouraged to use the production release of Sourcemash available at \url{www.sourcemash.com}. The website uses a Material Design framework\cite{Materialize} to create an easy-to-navigate, responsive user experience. The navigation menu, pinned to the left side of the page, is the primary tool used to access the site's core features and pages, which are explained in detail below:
 
-\subsection{Development: localhost}
+\begin{itemize}
+	\item \textbf{Browse} (\url{sourcemash.com/browse}): The Browse page contains an offering of feeds that users can subscribe to. If users do not see a feed that they wish to add, they may add their own using the form at the top of the page.
+	\item \textbf{Categorizer} (\url{sourcemash.com/categorizer}): This page offers standalone access to the Sourcemash categorization engine. The categorizer allows users to input the URL of any online article and returns auto-generated categories based on our categorization algorithm.
```

Comment:
"The categorizer allows users to input the URL of any online article and returns auto-generated categories based on our categorization algorithm." --> "The categorizer allows users to input the URL of any online article and receive auto-generated categories based on our categorization algorithm."